checkov/terraform/checks/resource/azure/StorageAccountName.py

```diff
@@ -21,7 +21,7 @@ def scan_resource_conf(self, conf):
         :param conf: azurerm_storage_account configuration
         :return: <CheckResult>
         """
-        return CheckResult.PASSED if conf.get('name') and re.match(STO_NAME_REGEX, conf['name'][0]) else CheckResult.FAILED
+        return CheckResult.PASSED if conf.get('name') and (re.match(STO_NAME_REGEX, conf['name'][0]) != None) else CheckResult.FAILED
```

Comment:
I would use 're.findall` for cleaner code. On case there's no match, it would yield an empty array

Reply:
I agree